Capstone project 1

kkolade opened the pull request:
The capstone project was built according to provided specification
- Personalize the content of your page. Choose a topic that is different than the one in the original design.

- Follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
  - Colors.
  - Typography: font face, size and weight 
  - Layout: composition and space between elements.
  - The pages should look almost identical to the original design

- You should build only these 2 pages:
  - The home page.
  - The about page.
- Each of these pages should have versions for 2 different screen sizes:
   - Mobile: up to 768px wide.
   - Desktop: 768px or wider.
-Interactions
   -Links
   - The home page should have a link in the menu to the about page.
   - The logo in the header links to the home page.
- Mobile menu
   - When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
   - There are no guidelines for the mobile menu in the docs, implement it so it is consistent with the design (colors, typography, spacings, etc.).
   - The mobile menu should have a close (X) button that closes the menu.
-Dynamic page
   -The section "Featured speakers" should be created dynamically in JavaScript.
   - A JavaScript variable with the data about the speakers should be created and use it when the page loads to create the HTML for this section dynamically.

kkolade committed:
Project Completed

kkolade committed:
Updated radme.md

kkolade committed:
Fix linter errors

kkolade committed:
Added title to the html pages

kkolade committed:
updated linters.yml

cynthiainga commented on README.md:
```diff
@@ -1 +1,242 @@
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
```
- [x] Please kindly make sure the project is deployed properly, and all pages, links, and images work as expected using the GitHub Pages. The link that you provided is not working, kindly fix that 🙃 🙃

cynthiainga commented on about.html:
```diff
@@ -0,0 +1,199 @@
+        </a>
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a href="about.html" class="header__nav-link">About</a>
```
- [x] Please kindly make sure it's possible to navigate between the pages, there's a link from the `Home` page to the `About` page and the opposite too. This means the nav menu of the about page should contain the link to the home page then users can navigate between the two pages easily, kindly fix that 🙃 😉

cynthiainga commented on about.html:
```diff
@@ -0,0 +1,199 @@
+  </head>
+  <body>
+    <header class="header">
+      <div class="header__user-bar">
+        <div class="header__user row">
+          <i class="fa-brands fa-facebook-f"></i>
+          <i class="fa-brands fa-twitter"></i>
+          <p class="header__language">English</p>
+          <p class="header__user-page">My page</p>
+          <p class="header__logout">Logout</p>
+        </div>
+      </div>
```
- [x] I think it can be better if you put this part into the `ul` tag and then each link into the `li` tag, also make sure the two social media links are in the `a` tag 😉 🙃. Do this change in the `index.html` file as well 👍

cynthiainga commented on about.html:
```diff
@@ -0,0 +1,199 @@
+        </div>
+      </div>
+    </section>
+    <footer class="footer" id="footer">
+      <div class="footer__container row">
+        <div class="footer__organizer">
+          <img
+            src="./build/images/sprintplex.jpeg"
+            alt=""
+            class="footer__organizer-logo"
+          />
+        </div>
+        <div class="footer__detail">
+          <p class="footer__copyright">
+            &copy;2022 Sprintplex Limited All rights reserved
+          </p>
+          <p class="footer__copy">
+            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Doloremque
+            molestias rerum eius saepe facere fugit labore perspiciatis odit
+            veniam dolor.
+          </p>
+        </div>
+      </div>
+    </footer>
```
- [x] As you can see in the design the footer of the about page has a dark background. Please kindly fix that 🙃 😉

Your design
<img width="1021" alt="Screen Shot 2022-12-02 at 02 12 45" src="https://user-images.githubusercontent.com/74501523/205178801-9c033ab8-45cd-4889-8ad4-4a4b34eca955.png">

Origin design
<img width="874" alt="Screen Shot 2022-12-02 at 02 13 06" src="https://user-images.githubusercontent.com/74501523/205178885-2fa2da81-0ca8-453e-ae5e-7d7cf9700e5b.png">

cynthiainga commented on about.html:
```diff
@@ -0,0 +1,199 @@
+            /></a>
+          </li>
+        </ul>
+        <div class="header__hamburger">
```
- [ ] The hamburger menu is on the left side, please kindly fix that 👍

Your design
<img width="469" alt="Screen Shot 2022-12-02 at 02 07 36" src="https://user-images.githubusercontent.com/74501523/205180852-8ba3be70-4f07-4fa3-ac9d-7825de8353ed.png">

Origin design
<img width="461" alt="Screen Shot 2022-12-02 at 02 29 17" src="https://user-images.githubusercontent.com/74501523/205180885-3fe4ae6a-763f-42a8-a6e8-641d420f11c2.png">

cynthiainga reviewed (changes requested):
Hi @kkolade, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://user-images.githubusercontent.com/80612925/157397398-00904e26-60ec-46cd-9510-240661c250ef.gif)

## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit history ✔️ 

## Required Changes ♻️

- [x] Please as mentioned in the project requirements you should record a video. Below you can see the requirements 👇 
<img width="1032" alt="Screen Shot 2022-12-02 at 01 34 15" src="https://user-images.githubusercontent.com/74501523/205173680-84864aec-8645-42a2-85d6-0791f8060b26.png">

- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

kkolade commented:
Thanks, @cynthiainga for your prompt yet thorough work on the review, I have worked on the issues raised and resolved them, but I must point out issues about the links between the home page and the about page, and also the mobile menu, and call your attention to the relevant sections of the requirement documentation. I believe I have implemented what the specification required in those issues. Once again thanks

kkolade committed:
Implemented issues raised by reviewer

cynthiainga commented:
> Thanks, @cynthiainga for your prompt yet thorough work on the review, I have worked on the issues raised and resolved them, but I must point out issues about the links between the home page and the about page, and also the mobile menu, and call your attention to the relevant sections of the requirement documentation. I believe I have implemented what the specification required in those issues. Once again thanks

Hello @kkolade, hope you are doing well. So about the issue between the two pages, I suggested you put the link to the home page into the about page because it's not all users can know if the logo direct to the home page, so if users navigate to the about page they can be in trouble on how to return to the home page. Having the link in the about page menu can allow everyone to navigate between the two pages easily 😊.
And about the hamburger menu as you can see in the original design the hamburger menu is on the left side. You can just put it to the left and then the logo to the right, you should stick as much as possible to the original design except the content(text, titles,...)

jerryowusu commented on about.html:
```diff
@@ -0,0 +1,168 @@
+            /></a>
+          </li>
+        </ul>
+        <div class="header__hamburger">
```
As requester by the previous reviewer, kindly put the hamburger menu to the left side in conformity with the given design. I know it is hard to accept but this is part of the project requirements which needs to be done before you can move to the next level. :pray:

jerryowusu commented on about.html:
```diff
@@ -0,0 +1,168 @@
+            class="header__brand-img"
+          />
+        </a>
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a href="about.html" class="header__nav-link">About</a>
```
Ask requested by the previous reviewer kindly make sure it's possible to navigate between the pages, there's a link from the Home page to the About page and vice versa. I can see that all the menu links of the mobile version goes to the about page including the logo at the bottom. Kindly consider changing maybe the links of the `Program` and logo to direct to the main page :pray:

kkolade commented:
@cynthiainga thanks will do that

jerryowusu reviewed (changes requested):
Hi @kkolade , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![so close](https://user-images.githubusercontent.com/65233860/205256485-2a02bfc5-e069-4b57-866a-6f811a221a5d.gif)


 ## Highlights
All linters are passing :heavy_check_mark: 
Excellent descriptive pull request summary :heavy_check_mark: 
Followed GitHub flow :heavy_check_mark: 

 ## Required Changes ♻️

- [ ] I am not able to see the project documentation video in your pull request description. :mag:  This may be an oversight. Kindly check and place it there since it is part of the project requirements as shown below :point_down: 

<img width="1032" alt="video" src="https://user-images.githubusercontent.com/65233860/205256456-61dcb2af-333a-49c4-b037-2936a8156157.png">


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

kkolade committed:
Fixed PR issues

Moved the hamburger to the left on mobile menu
added home link to the nav on about page

kkolade committed:
fix mobile menu as per PR

kkolade commented:
Hi @cynthiainga and @jerryowusu thanks for your great job,
- The video link was attached in the last submission, anyway please find it below:
https://www.loom.com/share/fb664ace339c4609a62b5e77f631ae50
- All the changes are implemented as per @cynthiainga request

Debas-31 reviewed (approved):
> ## Hi @kkolade  :wave:, 

# Status: Approved :heavy_check_mark:  :heavy_check_mark:  :heavy_check_mark: 
Great work on making the changes requested by a previous reviewer. Your project is complete !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
<img src="https://media1.giphy.com/media/M9HnDXluCEgMWtqkPv/giphy.gif"/>

## Highlights
- Linters setup :heavy_check_mark:
- GitHub flow :heavy_check_mark: 
- PR title and summary :heavy_check_mark: 
- Commit message :heavy_check_mark:
- UI design :heavy_check_mark: 
- Project functionality :heavy_check_mark: 
- Template usage :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
